pipeline_steps/common.groovy

```diff
@@ -516,33 +516,48 @@ def docker_cache_workaround(){
    sh "touch -t 201704100000 *.txt"
 }
 
-def is_doc_update_pr(String git_dir) {
-  if (env.ghprbPullId != null) {
-    dir(git_dir) {
+/* Used to check whether a PR only includes changes
+ * which are defined in the skip list.
+ * Expects a list of regex items to provide to egrep
+ * and returns a boolean.
+ */
+def skip_list_test(String[] args) {
```

Comment:
`def` can be changed to `Boolean` to both document and enforce that it returns a boolean value.

Reply:
Nice idea - will do.